src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs

```diff
@@ -85,13 +85,14 @@ public static async Task<IEnumerable<SuggestedActionSet>> WaitForItemsAsync(Test
             }
             catch (OperationCanceledException) when (!cancellationToken.IsCancellationRequested)
             {
-                if (Version.Parse("17.1.31928.29") >= await testServices.Shell.GetVersionAsync(cancellationToken))
+                var version = await testServices.Shell.GetVersionAsync(cancellationToken);
+                if (Version.Parse("17.2.32127.420") >= version)
```

Comment:
@AmadeusW Heads up that async light bulb is still dismissing randomly without request